### DISCUSSION TITLE
Many strange parent spanId

### DISCUSSION THREAD

**Discussion by lmssy** [open-telemetry/opentelemetry-java-instrumentation#3467] [Q&A]
Hi, I use opentelemetry-java-instrumentation with javaagent. 
I found many strange parent spanId which value is "//////////8=" in many different traces at different time.
But when I search "//////////8=" as spanId, I can't find any trace.
"//////////8=" seems not a regular spanId nor parentSpanId

Need your suggestion, thx in advance!

![image](https://user-images.githubusercontent.com/17894328/124089465-7f3b5600-da86-11eb-9e8c-4ec619cd33b3.png)

**Comment by trask** [comment]
hi @lmssy! what backend system is that screenshot from? my initial guess is maybe it's an odd rendering for the "no parent" parentSpanId "0000000000000000"

**Reply by lmssy** [reply]
Hi @trask ,backend system is elasticsearch

**Reply by trask** [reply]
Interestingly `SpanId.fromBytes(Base64.getDecoder().decode("//////////8="))` is `ffffffffffffffff`. I think you'll need to ask your backend system if that's a special value they use for "no parent". Typically no parent would be `0000000000000000`.
